packages/strapi-design-system/webpack.config.js

```diff
@@ -2,19 +2,27 @@ const fs = require('fs');
 const path = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
+const distPath = path.join(__dirname, 'dist');
+const isV2Build = process.env.IS_V2 === 'true';
+const outputPath = isV2Build ? path.join(distPath, 'v2') : distPath;
+
 // Allows to create distinct bundles in the dist folder
 // for people wanting to import only specific components such as
 // import Button from '@strapi/design-system/Button
-const excludedFolders = ['helpers', '.DS_Store'];
-const fileNames = fs.readdirSync(path.resolve(__dirname, 'src'));
+const entryFolder = isV2Build ? path.join(__dirname, 'src', 'v2') : path.join(__dirname, 'src');
+const excludedFolders = isV2Build ? ['helpers', '.DS_Store'] : ['helpers', '.DS_Store', 'v2'];
```

Comment:
Can you create a util here and reuse it in both the createIndexFiles.js and the webpack config?